src/controllers/softtakeover.cpp

```diff
@@ -94,11 +101,11 @@ bool SoftTakeover::ignore(ControlObject* control, double newParameter) {
         const double prevDiff = currentParameter - m_prevParameter;
         if ((prevDiff < 0 && difference < 0) ||
```

Comment:
Even though the logic for ==0 is never tested, because diffs at zero are never greater than the threshold I think this logic is actually ok